src/AutoRest.CSharp/Common/AutoRest/Communication/StandaloneGeneratorRunner.cs

```diff
@@ -263,6 +272,8 @@ internal static void LoadConfiguration(string? projectPath, string outputPath, s
                 projectPath ?? ReadStringOption(root, Configuration.Options.ProjectFolder),
                 protocolMethods,
                 suppressAbstractBaseClasses,
+                treatEmptyStringAsNullForModels,
+                treatEmptyStringAsNullForTypes,
```

Comment:
better to align with the same name:
```suggestion
                treatEmptyStringAsNullForExtraTypes,
```

Reply:
ah, so here you make a union of models and extra types

Reply:
Yeah, users set the extra types through autorest.md configuration, but we save the full list to `Configuration.json` for less surprise.